urllib3/util/ssl_.py

```diff
@@ -195,6 +197,16 @@ def create_urllib3_context(ssl_version=None, cert_reqs=ssl.CERT_REQUIRED,
     """
     context = SSLContext(ssl_version or ssl.PROTOCOL_SSLv23)
 
+    if hasattr(context, 'urllib3_wrapper_context'):
+        warnings.warn(
+            'A true SSLContext object is not available. This prevents urllib3 '
+            'from configuring SSL appropriately. This may cause certain SSL '
```

Comment:
two "This"'s in a row is a little awkward, how about "from configuring SSL appropriately and may cause certain SSL connections to fail"